### DISCUSSION TITLE
Tracker tramite SSID Wifi

### DISCUSSION THREAD

**Discussion by BaXaras** [indomus/forum#1873] [HUB: Home Assistant]
Buonasera, vi spiego il problema io e mia moglie utilizziamo l'app di HA io ho sempre il GPS attivo e HA riesce a vedere sempre ed esattamente che sono in casa o fuori mia moglie non ha attivo il GPS e HA la vede sempre in casa, voi ovviamente mi direte di a tua moglie di attivare il GPS e risolvi tutto, ma!! Come posso cambiare il suo stato in base allo stato del suo WIFI?

**Comment by 50494554524F** [comment]
basta controllare che sia attaccata al router di casa con "device_tracker" oppure provare a pingare l'ip locale

**Reply by Scialla** [reply]
```# Example configuration.yaml entry
device_tracker:
  - platform: ping
    hosts:
      device_name_1: 192.168.2.10
```
ovvio che devi dare ip statico al telefono di tua moglie
